libs/ui/src/lib/timed-notice/timed-notice.component.html

```diff
@@ -0,0 +1,54 @@
+<div
+  class="cvi-timed-notice__container cvi-timed-notice__container--severity-{{
+    severity
+  }}"
+>
+  <div class="cvi-timed-notice__icon-container">
+    <cvi-ng-icon [name]="getIconName()" [height]="24"></cvi-ng-icon>
+  </div>
+  <div class="cvi-timed-notice__content">
+    <cvi-ng-track
+      layout="flex"
+      horizontalAlignment="justify"
+      verticalAlignment="center"
+      [flexIsMultiline]="true"
+    >
+      <cvi-ng-track
+        layout="flex"
+        horizontalAlignment="left"
+        verticalAlignment="center"
+        [gap]="4"
+      >
+        <cvi-ng-track
+          layout="flex"
+          horizontalAlignment="center"
+          verticalAlignment="top"
+          flexDirection="vertical"
+        >
+          <div class="cvi-timed-notice__title">{{ dueDate }}</div>
+          <div>{{ time }}</div>
+        </cvi-ng-track>
+        <cvi-ng-track
+          flexDirection="vertical"
+          layout="flex"
+          horizontalAlignment="center"
+          verticalAlignment="top"
+        >
+          <div class="cvi-timed-notice__title underlined">
```

Comment:
The title is underlined and has a "clickable" pointer but is not actually an interactive element (button/link). 
End users may be confused as well seeing that the element affords clicking but does not do anything. Maybe this could be discussed with the designer of the Figma view?